lib/awful/layout/init.lua

```diff
@@ -123,14 +123,19 @@ function layout.arrange(screen)
         p.workarea = capi.screen[screen].workarea
         local useless_gap = tag.getgap(tag.selected(screen),
                                        #client.tiled(screen))
+        local ugap_edge = 0
+        local ugap_d = math.floor(useless_gap/2)
```

Comment:
Nope, because we have compensation `upag_m=useless_gap mod 2`, wich will add to right or bottom client edge if useless_gap is odd